krun/scheduler.py

```diff
@@ -84,6 +86,7 @@ def _parse(self):
         self._reset()
         fh = self._open()
         offset = 0
+        seen_headers = []
```

Comment:
Yes, but that's less efficient (and less obvious). Might as well bite the bullet upfront!